### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by aheyne**
Signed-off-by: Austin Heyne <aheyne@ccri.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,159 @@
-# GeoMesa Ingest with Nifi
+GeoMesa NiFi Plugins
```
Comment (jahhulbert-ccri) [inline README.md]: rename this "GeoMesa NiFi Bundle"

File: README.md [hunk 2]
```diff
@@ -1,43 +1,159 @@
...
 
-start up nifi
-cd $NIFI_HOME
-bin/nifi.sh start
+```bash
+$ mvn clean install
+```
 
-tail -f logs/nifi-app.log
+To install the GoeMesa processors you will need to copy the geomesa-nifi-nar file from
```
Comment (jahhulbert-ccri) [inline README.md]: spelling of geomesa wrong

File: README.md [hunk 3]
```diff
@@ -1,43 +1,159 @@
...
+``geomesa-nifi/geomesa-nifi-nar/target/geomesa-nifi-nar-$VERSION.nar`` into the ``lib/`` directory of your NiFi installation.
 
-Go to http://localhost:8080/nifi
+### Install the SFTs and Converters
 
-Drag a GeoMesaIngestProcessor down...fill out the accumulo instance, etc...for sft and converter try "example-csv" and 
-copy an example csv from the geomesa-tools/examples/ingest/csv folder into a temp dir...use the getfile processor to 
-pick it up and route it to the PutGeoMesa...
+The GeoMesa processors need access to the SFTs and converters in order to ingest data. We can provide these to the
```
Comment (jahhulbert-ccri) [inline README.md]: I'd probably mention here that you can do one of two things....install this or just copy/paste them directly into the processor's configuration which is often easier if you dont' want to install or update the jar.

File: README.md [hunk 4]
```diff
@@ -1,43 +1,159 @@
...
+``ControllerServices`` tab and click the ``+`` to add a new service. Search for the ``GeomesaConfigControllerService``
+and click add. Edit the new service and enter the appropriate values for the properties listed.
+
+To use this feature, after configuring the service, select the appropriate Geomesa Config Controller Service from the drop down
+of the ``GeoMesa Configuration Service`` property. When a controller service is selected the ``zookeepers``, ``instanceId``,
+``user``, ``password`` and ``tableName`` parameters are not required or used.
+
+### PutGeoMesaKafka
+
+The ``PutGeoMesaKafka`` processor is used for ingesting data into a Kafka backed GeoMesa datastore. To use this processor
```
Comment (jahhulbert-ccri) [inline README.md]: prob should mention something about what version of kafka it uses

File: README.md (hunk 1 above)
Comment (aheyne) [inline README.md]: fixed

File: README.md (hunk 2 above)
Comment (aheyne) [inline README.md]: fixed, nice catch!

File: README.md (hunk 3 above)
Comment (aheyne) [inline README.md]: added.

File: README.md (hunk 4 above)
Comment (aheyne) [inline README.md]: added

**Pull request merged**